ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6605,4 +6605,52 @@ VMADC:
   - mixed_air_damper_percentage_command
   - mixed_air_damper_percentage_sensor
   implements:
-  - CONTROL
\ No newline at end of file
+  - CONTROL
+
+SFPC:
+  description: "supply pressure monitoring, flow control."
+  is_abstract: true
+  opt_uses:
+  - supply_fan_run_command
+  - supply_air_linearvelocity_sensor
+  - supply_fan_rotationalvelocity_sensor
+  uses:
+  - supply_air_static_pressure_sensor
+  - supply_air_flowrate_sensor
+  - supply_air_flowrate_setpoint
+  implements:
+  - OPERATIONAL
+
+EFPC:
+  description: "exhaust pressure monitoring, flow control."
+  is_abstract: true
+  opt_uses:
+  - exhaust_fan_run_command
+  - exhaust_fan_rotationalvelocity_sensor
+  uses:
+  - exhaust_air_flowrate_sensor
+  - exhaust_air_flowrate_setpoint
+  - exhaust_air_static_pressure_sensor
+  implements:
+  - OPERATIONAL
+
+ESDC:
```

Comment:
Can we use a combination of `EDM` and `SDM`?

Reply:
yes, changed